jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/MetaData.java

```diff
@@ -757,7 +762,34 @@ public boolean isValidateXml()
      */
     public void setValidateXml(boolean validateXml)
     {
-        _validateXml = validateXml;
+        if (_xmlParser != null)
```

Comment:
But this class always instantiates an instance of `XmlParser`, so it will never be null, so you can never get a validating parser by calling `setValidateXml(true)`.